README.md

```
@@ -1,115 +1,645 @@
-# Testing
-pnpm exec playwright test              # Run tests
-pnpm exec playwright test --ui         # Run tests in UI mode
+- **Pre-commit**: Auto-format and lint staged files
+- **Commit**: Validates commit messages
+- **Pre-push**: Runs type checks
+
+---
+
+## 📂 Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)